prov/tcp/src/tcpx.h

```diff
@@ -84,6 +84,7 @@ struct tcpx_xfer_entry;
 struct tcpx_cq;
 struct tcpx_ep;
 struct tcpx_rx_detect;
+enum tcpx_xfer_op_codes;
```

Comment:
nit: can move the enum declaration up in the file to avoid the forward declaration